open_instruct/finetune.py

```diff
@@ -545,8 +550,12 @@ def main(args: FlatArguments):
         experiment_config = vars(args)
         # TensorBoard cannot log Enums, need the raw value
         experiment_config["lr_scheduler_type"] = experiment_config["lr_scheduler_type"]
+        if args.wandb_entity is None:
+            args.wandb_entity = maybe_use_ai2_wandb_entity()
         accelerator.init_trackers(
-            "open_instruct_sft", experiment_config, init_kwargs={"wandb": {"entity": args.wandb_entity}}
+            "open_instruct_internal",
+            experiment_config,
+            init_kwargs={"wandb": {"entity": args.wandb_entity, "tags": [os.path.basename(__file__)[: -len(".py")]]}},
```

Comment:
Should we hardcode the tags instead of doing the python pathing that may become broken in future file moving?

Reply:
I was hardcoding initially, but decided to do it this way to reduce lines of code differences. Then, we also don't have to do this for different files (e.g., `ppo.py`, `online_dpo.py`)

<img width="1364" alt="image" src="https://github.com/user-attachments/assets/c1948baa-3238-417c-b7cc-4f022628f76f">